src/DependencyInjection/PimcoreQrcodeExtension.php

```diff
@@ -2,25 +2,55 @@
 
 namespace GalDigitalGmbh\PimcoreQrcodeBundle\DependencyInjection;
 
+use GalDigitalGmbh\PimcoreQrcodeBundle\Model\QrCode\Dao;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\HttpKernel\DependencyInjection\ConfigurableExtension;
 
-class PimcoreQrcodeExtension extends Extension
+class PimcoreQrcodeExtension extends ConfigurableExtension implements PrependExtensionInterface
 {
     /**
-     * @param array<mixed> $configs
+     * @param array $mergedConfig
```

Comment:
```suggestion
     * @param array<mixed> $mergedConfig
```

Reply:
You can run `php vendor/bin/phpstan analyze src --level=8 --memory-limit=-1` to check for static analysis issues